colibre/config.yml

```diff
@@ -156,27 +163,42 @@ scripts:
     title: Number of steps vs. cosmic time
     section: Run Performance
     output_file: simulation_time_number_of_steps.png
+  - filename: scripts/density_temperature_dust2metal.py
+    caption: Density-temperature diagram shaded by the total fraction of metals in the dust phase.
+    output_file: density_temperature_dust2metal.png
+    section: Density-Temperature
+    title: Dust-to-metal Ratio
+    additional_arguments:
+      quantity_type: hydro
+  - filename: scripts/density_species_interp.py
+    caption: Co-plot of species fractions (left y-axis) and the dust-to-metal ratio (right y-axis) as a function opf gas density. If available, solid green line shows explicitly modelled dust-to-metal ratio, while dashed green is interpolated from the Ploeckinger+20 tables.
+    output_file: density_vs_species_fraction.png
+    section: Hydrogen Phases
+    title: Hydrogen Phase Fractions
+    additional_arguments:
+      cooling_tables: /cosma7/data/dp004/wmfw23/colibre_dust/coolingtables/UV_dust1_CR1_G1_shield1.hdf5
+      quantity_type: hydro
   - filename: scripts/density_temperature_species.py
-    caption: Density-temperature diagram shaded by H$_2$ mass fraction. The fraction is computed as the H$_2$ mass contained in each cell over the mass of gas in that cell.
+    caption: Density-temperature diagram shaded by H2 mass fraction.
```

Comment:
Can we keep the latex-ing in these, please? So H$_2$. I was also requested to add that additional comment, so if it's incorrect can we replace it with a correct alternative?

Reply:
added these back, removed accidentally